modules/openxr/scene/openxr_composition_layer.cpp

```diff
@@ -103,21 +123,27 @@ void OpenXRCompositionLayer::_create_fallback_node() {
 	should_update_fallback_mesh = true;
 }
 
+void OpenXRCompositionLayer::_remove_fallback_node() {
+	ERR_FAIL_COND(fallback);
```

Comment:
Should be `ERR_FAIL_NULL`

Reply:
This is the opposite condition, should probably be made more clear with:
```suggestion
	ERR_FAIL_COND(fallback != nullptr);
```